samples/suit/recovery/README.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
 .. _suit_recovery:
 
 SUIT recovery application
```

Comment:
I assume there should also be a user guide for recovery...  to a large extent it would contain similar contents to this file, yet the sample is not really the best place for this whole documentation.

Should we create a placeholder so that we manage to introduce it for 2.7.0